branding/pdf/entities.adoc

```diff
@@ -44,6 +44,7 @@
 :suse: SUSE
 :slea: SLE
 :sleda: SLED
+:phub: '{suse} Package Hub'
```

Comment:
I'm not sure about the apos.

Reply:
they're required if there's a space in the expanded entity. i discovered this the hard way :wink:

Reply:
Thanks for background info!  If I'll need it, it will save me time!